.github/workflows/codespell.yml

```diff
@@ -0,0 +1,17 @@
+---
+name: Codespell
+
+on:
+  push:
```

Comment:
Leave only pull_request and develop+master branches

Reply:
a little unclear to me since that is what it is now -- pull requests (against develop and master) and pushes (so the branches themselves).  Leaving only pull requests would be IMHO suboptimal since if any direct push is done to any of those branches -- would be left "untested".
Please clarify, or ideally provide `suggestion`

Reply:
Okay, I agree with you here.